### PR TITLE
Created Splash Page:

### PR DESCRIPTION
**Pull request by stecd**
1) Oliver wrote a new index and css file
2) Stephanie edited some of oliver's code so that it worked more responsively
4) updated video background to have no text
3) Twitter link still has to be updated

### CODE REVIEW

File: index-0.html [hunk 1]
```diff
@@ -0,0 +1,254 @@
+<!--
```
Comment (tinataleb) [inline index-0.html]: can you rename this file to something like "v1index" so we know this was the first version?

File: index.css [hunk 2]
```diff
@@ -0,0 +1,211 @@
+#content {
```
Comment (tinataleb) [inline index.css]: Can you move this file to assets/css?

File: index.html [hunk 3]
```diff
@@ -31,224 +18,44 @@
   <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:400,700|Material+Icons" />
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/latest/css/font-awesome.min.css">
   <!-- CSS Files -->
+  <link href="index.css" rel="stylesheet" />
```
Comment (tinataleb) [inline index.html]: After moving index.css to assets/css this has to be updated.

**Comment by tinataleb** [comment]
The animation plays on a loop, my impression was that it should play once and stop at the end without going dark. 
Is this an expected behavior?

**Comment by D-Arceneaux** [comment]
added in the C19.png photo for the web browser photo integration.
Code is there, .png missing

**Comment by stecd** [comment]
@tinataleb yeah, Dylan and I liked the looping so we left it. but feel free to change it!

**Pull request merged**